FirebaseAnalyticsOnDeviceConversion.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name             = 'FirebaseAnalyticsOnDeviceConversion'
-    s.version          = '9.1.0'
+    s.version          = '9.2.0'
```

Comment:
Please update line 21 to instead be:
```ruby
s.dependency 'GoogleAppMeasurementOnDeviceConversion', '9.2'
```

Reply:
Related to this discussion: https://github.com/firebase/firebase-ios-sdk/pull/9787#discussion_r868524192

Reply:
Done.

Reply:
Thank you, to fix the CI, could you now pod repo push the `FirebaseAnalyticsOnDeviceConversion.podspec` (you can copy and paste it from the command that pushed it earlier).

Reply:
Done. Using the cmd in the doc.

Reply:
ty! [Looks good!](https://github.com/firebase/SpecsStaging/commit/fb262a1aaa6c8a84381d08798883552204d3f298) Sorry for the extra steps.